sql/commands/sql-create-mv.mdx

```diff
@@ -29,6 +27,48 @@ Names and unquoted identifiers are case-insensitive. Therefore, you must double-
 The `ORDER BY` clause in the `CREATE MATERIALIZED VIEW` statement is allowed but not considered as part of the definition of the materialized view. It's only used in the initial creation of the materialized view and not during refreshes.
 </Note>
 
+## Backfill behavior and controls
+
+1. `CREATE MATERIALIZED VIEW` will first **backfill** historical data from the referenced relations, and completion time varies based on the volume of data to be backfilled.
+
+2. To perform the operations in the background, you can execute `SET BACKGROUND_DDL=true;` before running the `CREATE MATERIALIZED VIEW` statement. See details in [SET BACKGROUND\_DDL](/sql/commands/sql-set-background-ddl).
+
+3. To avoid join amplification during backfill, you can optionally specify `backfill_order` in the `WITH` clause to control the order. Use the `->` operator to define dependencies between tables, and the left table will be fully backfilled before the right begins.
+
+    ```sql
+    CREATE MATERIALIZED VIEW m1
+    WITH (backfill_order = FIXED(t1 -> t2, t2 -> t3))
+    AS
+    SELECT v1 FROM t1
+    UNION
+    SELECT v1 FROM t2
+    UNION
+    SELECT v1 FROM t3;
+    ```
```

Comment:
In the above example, we will have the following sequence of backfill:
1. t1
2. t2
3. t3